compose-files/docker-compose-delhi-0.7.1-arm64.yml

```diff
@@ -0,0 +1,202 @@
+# /*******************************************************************************
+#  * Copyright 2018 Dell Inc.
```

Comment:
Is this still Copyright Dell? Did you write this yourself or did you modify a different compose file? I see you have author set below in the comment, thanks for that

Reply:
Actually, I forgot to delete that comment. I used the docker-compose as boilerplate but the configuration is my creation, so that comment should be removed. 

First time doing a PR, so not sure how to modify it.. :P

Reply:
You just need to push a new commit to the same branch in Github and it will update this PR automatically